src/js/components/Menu.js

```diff
@@ -24,7 +25,8 @@ var MenuDrop = React.createClass({
     large: React.PropTypes.bool,
     onClick: React.PropTypes.func.isRequired,
     router: React.PropTypes.func,
-    small: React.PropTypes.bool
+    small: React.PropTypes.bool,
+    navContainerId: React.PropTypes.string.isRequired
```

Comment:
The same ID must be know by Menu and MenuDrop. The way I found was creating it on Menu and then passing it to MenuDrop. I accept suggestions =)